terraform/aws/components/nhais_responder/locals_secret_variables.tf

```diff
@@ -0,0 +1,44 @@
+locals {
+    secret_variables = [
+    {
+      name = "NHAIS_AMQP_USERNAME"
+      valueFrom = data.aws_secretsmanager_secret.mq_username.arn
+    },
+    {
+      name = "NHAIS_AMQP_PASSWORD"
+      valueFrom = data.aws_secretsmanager_secret.mq_password.arn
+    },
+    {
+      name = "NHAIS_MONGO_USERNAME"
+      valueFrom = data.aws_secretsmanager_secret.docdb_master_username.arn
+    },
+    {
+      name = "NHAIS_MONGO_PASSWORD"
+      valueFrom = data.aws_secretsmanager_secret.docdb_master_password.arn
+    },
+    {
+      name = "NHAIS_RESPONDER_MESH_MAILBOX_ID"
```

Comment:
As above NHAIS_RESPONDER... prefix is not used any any of the environment variable names. ENV names are the same as for NHAIS. Not going to comment on each of them below but they all need fixing.